### PR TITLE
feat: Update commit status with CI job result

### LINKED ISSUE #5: feat: Notify user when CI Server has completed task
The CI server can sets the commit status on the Github repository.
This seems pretty straight forward, and saves us from being bombarded with emails.
Example:
Sending a POST request with a specified commit looks like this:
```
curl -X POST -H 'Content-Type: application/json' --data 
'{"state":"success","description”:”Build passed“,”context”:”Dump CI”}’ 
https://fed5dd5e61b6b76afc9cb276f426d03dc026eaac:x-oauth-basic@api.github.com/repositories/168348331/statuses/8ed426c83dc5ffda7909f9346bf626b35dfeec7b
```

Resources:

Git API Statues: https://developer.github.com/v3/repos/statuses/

Curl: https://stackoverflow.com/questions/18204326/how-to-post-url-in-data-of-a-curl-request

Set Git status with curl: https://stackoverflow.com/questions/40237009/github-statuses-examples

I've created a user @Dump-CI that has an OAuth-token: fed5dd5e61b6b76afc9cb276f426d03dc026eaac
Thus the CI server can use this token to set statuses on commits

### PR DESCRIPTION
**Pull request by lilfelix**
This PR implements the function`reportResult(GitEvent event)` in the class `CIWorker`.
The function makes a POST-request to the Github API to update the status of a commit on Github.
The commit is identified by `event.sha` and the status by `event.status`.

This PR is part Closes #5

### CODE REVIEW

**Comment by lilfelix** [comment]
Just realized I haven't written any tests for this. I'll add tests before re-opening the PR

**Comment by lilfelix** [comment]
I've added unit tests and created a separate issue (#17) for end-to-end testing of the functionality added in this PR

**Comment by fredrikfloven** [comment]
Ficed merge conflicts

File: src/main/java/CIWorker.java [hunk 1]
```diff
@@ -1,61 +1,182 @@
-import java.io.File;
-import java.io.IOException;
+import org.json.simple.JSONObject;
+
+import java.io.*;
```
Comment (fredrikfloven) [inline src/main/java/CIWorker.java]: Any reason for importing all io subclasses?

Comment (lilfelix) [reply src/main/java/CIWorker.java]: You're right that's unnecessary. Pushed a fix for it

**Pull request merged**